src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1209,7 +1209,7 @@ private void updateOverallHappiness()
             final BuildingHome home = citizen.getHomeBuilding();
             if(home != null)
             {
-                housing = home.getBuildingLevel();
+                housing += home.getBuildingLevel();
```

Comment:
That doesn't address the fact that a house that has been designed to hold 2 people, will have a maximum happiness of 2.5 at level 5.
Am I missing the reason we don't count the second person?